### ISSUE TITLE
Split build and run stages in dockerfile

### ISSUE THREAD

**Issue by shawaj** [NebraLtd/hm-gatewayrs#18]
Split build

**Comment by pritamghanghas** [comment]
Not sure this is required. There is no build step, atleast right now. We only install the deb package, those are anyway difficult to copy from builder.

**Comment by shawaj** [comment]
yep - for now there is no value in this i think so probably we can close?
